web/packages/teleport/src/DesktopSession/useDesktopSession.tsx

```diff
@@ -72,38 +71,20 @@ export default function useDesktopSession() {
     };
   }, []);
 
-  const [showAnotherSessionActiveDialog, setShowAnotherSessionActiveDialog] =
-    useState(false);
-
-  document.title = useMemo(
-    () => `${username}@${hostname} • ${clusterId}`,
-    [clusterId, hostname, username]
-  );
-
+  //TODO(gzdunek): This is workaround for synchronizing *sharingState with aclAttempt.
+  //Refactor clipboard and directory sharing so that we won't need allowedByAcl fields in state.
   useEffect(() => {
-    run(() =>
-      Promise.all([
-        desktopService
-          .fetchDesktop(clusterId, desktopName)
-          .then(desktop => setHostname(desktop.name)),
```

Comment:
This call was removed completely. It doesn't seem to do anything useful as it expects a desktop name and then returns it back.